docs/pages/overview/5-ideology.md

```diff
@@ -8,61 +8,61 @@ permalink: /overview/ideology.html
+GRAKN.AI is a fully integrated knowledge-base environment, which has storage, querying, validation, reasoning, IDE (in progress), visualisation and discovery (in progress) all in one system where the user does not need to care about integrations. RDF/OWL platforms require complex layers of loosely coupled components from many different systems, such as schema editors (e.g. Protege), storage (RDF triple stores), query engines (e.g. SPARQL), reasoners (e.g. Pellet, other OWL reasoners).
 
 ### Automated validation and reasoning
-Given an ontology, GRAKN.AI automatically takes care of validating input data in real-time, and reasoning/inference is a native behaviour of the Graql language. You immediately reap the validation and reasoning benefits of the ontology. It is not as straightforward in the RDF/OWL world; validation does not happen in real-time and reasoners are not naturally integrated. Validation is thus a slow, off-line computation and reasoning requires integration of separate reasoner-tools with your SPARQL query language, which is not always seamless, often very buggy, and does not scale.
+Given a schema, GRAKN.AI automatically takes care of validating input data in real-time, and reasoning/inference is a native behaviour of the Graql language. You immediately reap the validation and reasoning benefits of the schema. It is not as straightforward in the RDF/OWL world; validation does not happen in real-time and reasoners are not naturally integrated. Validation is thus a slow, off-line computation and reasoning requires integration of separate reasoner-tools with your SPARQL query language, which is not always seamless, often very buggy, and does not scale.
 
 ## Why are Schema-Constraints Necessary?
-GRAKN.AI’s ontology is basically an intelligent type system for your complex data.
-A complex relationship model combined with no data schema-constraint means that there are an exponential number of possible mistakes. Thus, one of value propositions of GRAKN.AI’s ontology is that it functions as a schema constraint to your data, which gets validated in real-time upon writing it to the database.
+GRAKN.AI’s schema is basically an intelligent type system for your complex data.
+A complex relationship model combined with no data schema-constraint means that there are an exponential number of possible mistakes. Thus, one of value propositions of GRAKN.AI’s schema is that it functions as a schema constraint to your data, which gets validated in real-time upon writing it to the database.
```

Comment:
now of the value propositions of our schema is that it works as a schema? That is AMAZING 🍌